Gruntfile.js

```diff
@@ -1,3 +1,4 @@
+/*eslint object-curly-newline: ["error", "always"]*/
```

Comment:
I would like to to keep this option and see how it works. It's easy to revert if we think it's not worth it but
>use inline enable/disable eslint comments for object literals we really think should be on one line
is something that won't work because I'd much rather have newlines than comments cluttering up the code